exporters/otlp/otlpmetric/otlpmetrichttp/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenTelemetry-Go OTLP Metric HTTP Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp)
+
+[OpenTelemetry Protocol Exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/protocol/exporter.md) implementation using HTTP with protobuf-encoded payloads.
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp
+```
+
+## Usage
+
+Exporters should be created using the [New](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp#New) and used with a [PeriodicReader].
+
+## Environment Variables
+
+The following environment variables can be used (instead of options objects) to override the default configuration.
+
+| Name | Description | Default | Override with |
+|------|-------------|---------|---------------|
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | Endpoint to which the exporter is going to send traces or metrics. If the scheme is `http` or `unix` this will also set `OTEL_EXPORTER_OTLP_INSECURE` to true |`http://localhost:4317`| `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT`, [WithEndpoint()] |
```

Comment:
```suggestion
| `OTEL_EXPORTER_OTLP_ENDPOINT` | Endpoint to which the exporter is going to send traces or metrics. If the scheme is `http` or `unix` this will also set `OTEL_EXPORTER_OTLP_INSECURE` to true | `http://localhost:4317` | `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT`, [WithEndpoint()] |
```